node-tests/acceptance/build-test.js

```diff
@@ -10,20 +10,12 @@ const build = require('../helpers/build');
 const InRepoAddon = require('../helpers/in-repo-addon');
 const InRepoEngine = require('../helpers/in-repo-engine');
 const expectedManifests = require('../fixtures/expected-manifests');
+const matchers = require('../helpers/matchers');
 
-/**
- * Creates a regex that matches the definition for the specified module name.
- *
- * @param {String} moduleName
- * @return {RegExp}
- */
-function moduleMatcher(moduleName) {
-  return new RegExp(`define\\(['"]${moduleName}['"]`);
-}
-
-function cssCommentMatcher(comment) {
-  return new RegExp(`/\\* ${comment} \\*/`);
-}
+const {
+  module: moduleMatcher,
+  cssComment: cssCommentMatcher
+} = matchers;
```

Comment:
Destructuring is not supported in node 4. I'm unsure if we run our test suite there though...